Sources/AsyncHTTPClient/HTTPHandler.swift

```diff
@@ -367,6 +367,12 @@ public class ResponseAccumulator: HTTPClientResponseDelegate {
         case error(Error)
     }
 
+    struct ResponseTooBigError: Error, CustomStringConvertible {
```

Comment:
We'll need this to be `public`.

Reply:
Ah, so that it can be handled by the (failed) future's consumer?

Reply:
Yeah, in general it's a good idea to make errors public if they're likely to be thrown.